okx/common/prv.go

```diff
@@ -155,7 +157,7 @@ func (prv *Prv) DoAuthRequest(httpMethod, reqUrl string, params *url.Values, hea
 		reqUri     string
 	)
 
-	if http.MethodGet == httpMethod {
+	if http.MethodGet == httpMethod && params.Encode() != "" {
```

Comment:
You can use OKX's simulated trading by adding 'options.WithSimulated()' or 'options.WithSimulated(true)'